.pre-commit-config.yaml

```diff
@@ -1,8 +1,9 @@
 repos:
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.2.2
+  rev: v0.3.3
   hooks:
     - id: ruff
+      args: [--fix]
```

Comment:
can you change this to `-diff` (is that exists for the `ruff` linter)? 
Sometimes the autofix fixes stuff we don't want to be changed so it's better to fail and show the diff.

Reply:
OK, sure. I've done that and the rebase. I'm positive that the checks will run :)